guides/hack/20-attributes/07-predefined-attributes.md

```diff
@@ -221,6 +221,8 @@ The types of the parameters are restricted to the following: `null`, `bool`, `in
 
 The interface type `IMemoizeParam` assists with memoizing objects passed to async functions.
 
+To clear the cache and avoid potential memory leaks, use `HH\clear_static_memoization`.
```

Comment:
This should not be done in production code; it's really only intended for unit tests, and only if the tests require breaking the isolation of 'a unit'. It shouldn't generally be used.

`<<__Memoize>>` is frequently abused as a 'run at most one time', so clearing caches can really break stuff.

Reply:
hmm do we have a way of embedding macro blocks? something like:
![image](https://user-images.githubusercontent.com/5179225/141000146-b6eeb01a-2fda-47d0-9e51-18c2233c0127.png)

Reply:
if you grep -i src/ for `fbonly` you'll see one special case of this, which could be generalized